### PR TITLE
fix website implementation

### PR DESCRIPTION
**Pull request by Neb19**
.gitignore and index.html files are modified.

### CODE REVIEW

**Comment by girard-g** [comment]
Well, you really shouldn't commit and push all generated files like all .css files or all .js files. We use bower, so we only need the bower configuration file cause all assets and dependences will be generated when we will execute.

`bower install`

See [this comment](https://github.com/UnderDevCrew/HapplyHourSite/pull/6#issuecomment-219254124)

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "directory" : "www/assets/bower_components"
+}
```
Comment (duncanroze) [inline .bowerrc]: :) eof

File: www/index.html [hunk 2]
```diff
@@ -55,42 +55,40 @@ <h2 class="text-center">What's Happly Hour ? </h2>
...
         <!-- Column -->
-        <div class="col-md-4">
+        <div class="col-sm-5">
             <h2 class="text-center">Find a bar</h2>
             <p class="text-center">
                 <span class="fa fa-hand-o-right"></span>
             </p>
             <p class="text-justify">
-                Just click on 'Bar Finder' and localize the path for to take a great Beer with your friends, who you want !
+                Just click on 'Bar Finder', find and localize the path for to take a good Beer with your friends, or who you want simply !
```
Comment (girard-g) [inline www/index.html]: Broken english bro

**Comment by pimolo** [comment]
https://libraries.io/bower/letterfx

File: gruntfile.js~ [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }
+  });
+
+  // Load plugins
+  require('load-grunt-tasks')(grunt);
+
+  // Default task(s).
+  grunt.registerTask('css-tasks', ['sass:dist', 'cssmin:dist']);
+
+};
```
Comment (pimolo) [inline gruntfile.js~]: Remove that

**Comment by pimolo** [comment]
http://bower.io/search/?q=parallax.js

**Comment by girard-g** [comment]
Good for me

**Comment by duncanroze** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/UnderDevCrew/HapplyHourSite/pull-request/7/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=UnderDevCrew/HapplyHourSite)

**Pull request merged**